tests/test_processor_composition.py

```diff
@@ -115,6 +115,27 @@ def test_remote_processor_creation(requests_mock):
     rp.add(0, BS())
 
 
+def test_remote_processor_composition_input_ports(requests_mock):
+    ls = LogicalState([0, 0])
+    cnot = catalog['postprocessed cnot'].build_processor()
+    cnot.min_detected_photons_filter(2)
```

Comment:
You can remove this line (you don't need it since you don't use `cnot.with_input(ls)`)